src/pomodoro/Timer.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect } from 'react';
+import TimerDigits from './TimerDigits';
+import { useTimerReducer } from './useTimerReducer';
+import IncreaseTimeContainer from './IncreaseTimeContainer';
+import Button from './Button';
+import TimerControls from './TimerControls';
+
+export default function Timer() {
+  const {
+    time,
+    isActivated,
+    timerSchema,
+    start,
+    stop,
+    reset,
+    decreaseTime,
+    increaseTimeByOneMin,
+    increaseTimeByFiveMin,
+    increaseTimeByTwentyMin,
+  } = useTimerReducer();
+
+  useEffect(() => {
+    if (isActivated) {
+      const intervalID = setInterval(decreaseTime, 1000);
+      return () => clearInterval(intervalID);
+    }
+  }, [isActivated]);
+
+  return (
+    <div>
+      <TimerDigits time={time} />
+
+      <IncreaseTimeContainer>
+        <Button
+          label='1'
+          onClick={increaseTimeByOneMin}
+          className='Button Button--type-square'
+        />
+        <Button
+          label='5'
+          onClick={increaseTimeByFiveMin}
+          className='Button Button--type-square'
+        />
+        <Button
+          label='20'
```

Comment:
Those labels are not very descriptive.

Reply:
I know, that I could name it something like add 1/5/20 minutes, but I thought that it would be better for design, if it has name '1/5/20'